ringpop_test.go

```diff
@@ -317,15 +318,15 @@ func (s *RingpopTestSuite) TestHandleEvents() {
 	s.Equal(int64(1), s.stats.read("ringpop.127_0_0_1_3001.refuted-update"), "missing refuted-update stat")
 
 	// double check the counts before the event
-	s.Equal(int64(11), s.stats.read("ringpop.127_0_0_1_3001.ring.server-added"), "incorrect count for ring.server-added before RingChangedEvent")
-	s.Equal(int64(2), s.stats.read("ringpop.127_0_0_1_3001.ring.server-removed"), "incorrect count for ring.server-removed before RingChangedEvent")
+	s.Equal(int64(9), s.stats.read("ringpop.127_0_0_1_3001.ring.server-added"), "incorrect count for ring.server-added before RingChangedEvent")
```

Comment:
There was an actual bug that caused servers that were already in the hashring to show up in the event that gets statted here. If there was at least 1 change made to the ring it would stat that all changes were made. With the new code it will only stat the actual servers added and removed from the ring. This caused the count to drop by 1.

Secondly there was a tests where 3 changes were made in rapid succession for the same node which is an unrealistic scenario which cannot happen and cannot be expressed easily with the new event. Causing an other drop in the `ring.server-added` stat.